tests/CommandLineTestRunnerTest.cpp

```diff
@@ -94,3 +95,25 @@ TEST(CommandLineTestRunner, NoPluginsAreInstalledAtTheEndOfARunWhenTheArgumentsA
 
     LONGS_EQUAL(0, registry.countPlugins());
 }
+
+struct TestOutputCheckingCommandLineTestRunner : public CommandLineTestRunner
+{
+    TestOutputCheckingCommandLineTestRunner(int ac, const char** av, TestOutput* output, TestRegistry* registry) :
+        CommandLineTestRunner::CommandLineTestRunner(ac, av, output, registry)
+    {
+    }
+
+    bool hasJUnitTestOutput(void)
+    {
+        return NULL != dynamic_cast<JUnitTestOutput*>(output_);
```

Comment:
It has an isJUnitOutput option :)

We don't use dynamic_casts in CppUTest due to backwards C++ compatibility.